model/accounts_mgmt/v1/related_resources_type.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+struct RelatedResources {
+	ResourceName         string
+	Byoc                 string
```

Comment:
`BYOC` is an acronym, but if you don't write it all uppercase then the code generator has no way to know that, and it will produce go names like `GetByoc` instead of `GetBYOC`. Not very important, but I think it would be better to explicitly write this all uppercase.